node.gyp

```diff
@@ -69,6 +69,8 @@
       'lib/v8.js',
       'lib/vm.js',
       'lib/zlib.js',
+
+      'lib/internal/freelist.js'
```

Comment:
Femto-nit: can you append a comma?  That makes the diff less noisy when new entries are added.